tests/test_chain.py

```diff
@@ -511,7 +511,7 @@ def test_retrieve_vault_status(revault_network, bitcoind):
     spend_psbt = serializations.PSBT()
     spend_psbt.deserialize(spend_tx)
     spend_psbt.tx.calc_sha256()
-    mans[0].rpc.setspendtx(spend_psbt.tx.hash)
+    mans[0].rpc.setspendtx(spend_psbt.tx.hash, False)
```

Comment:
In commit " tests: Pass priority in setspendtx and... ...in broadcast/spend utilities"

Why is it needed if you made the field optional on the RPC interface?

Reply:
Forgot to update after API update 😭